### PR TITLE
Update Netskope readme file 

### PR DESCRIPTION
**Pull request by cherryleaf**
Type of change

- Enhancement

## What does this PR do?

Tidy up of Markdown in Readme.md, including adding link to developer's documentation, if missing

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ]

## Related issues

-

### CODE REVIEW

**Review by mtojek — CHANGES_REQUESTED** [review]
Could you please revert changes in the `go.mod`? It shouldn't be part of this PR.

File: packages/netskope/_dev/build/docs/README.md [hunk 1]
```diff
@@ -10,32 +10,32 @@ ECS fields where applicable and the remaining fields are written under
...
-    For detailed steps refer [_Manage Log Shipper Business Rules_](https://docs.netskope.com/en/manage-log-shipper-business-rules.html).
+    For detailed steps refer to [Manage Log Shipper Business Rules](https://docs.netskope.com/en/manage-log-shipper-business-rules.html).
 8. Adding SIEM mappings:
     1. Navigate to Home Page > Log Shipper > SIEM Mappings
     2. Add SIEM mapping for events: 
         * Add **Rule** put rule created in step 7.
         * Add **Source Configuration** put input created for Events in step 4.
         * Add **Destination Configuration**, put output created for Events in step 6.
-    For detailed steps refer [_Configure Log Shipper SIEM Mappings_](https://docs.netskope.com/en/configure-log-shipper-siem-mappings.html).
+        For detailed steps refer to[Configure Log Shipper SIEM Mappings](https://docs.netskope.com/en/configure-log-shipper-siem-mappings.html).
```
Comment (bmorelli25) [inline packages/netskope/_dev/build/docs/README.md:38]: Missing a space

```suggestion
        For detailed steps refer to [Configure Log Shipper SIEM Mappings](https://docs.netskope.com/en/configure-log-shipper-siem-mappings.html).

```

File: packages/netskope/docs/README.md [hunk 2]
```diff
@@ -10,32 +10,32 @@ ECS fields where applicable and the remaining fields are written under
...
-    For detailed steps refer [_Manage Log Shipper Business Rules_](https://docs.netskope.com/en/manage-log-shipper-business-rules.html).
+    For detailed steps refer to [Manage Log Shipper Business Rules](https://docs.netskope.com/en/manage-log-shipper-business-rules.html).
 8. Adding SIEM mappings:
     1. Navigate to Home Page > Log Shipper > SIEM Mappings
     2. Add SIEM mapping for events: 
         * Add **Rule** put rule created in step 7.
         * Add **Source Configuration** put input created for Events in step 4.
         * Add **Destination Configuration**, put output created for Events in step 6.
-    For detailed steps refer [_Configure Log Shipper SIEM Mappings_](https://docs.netskope.com/en/configure-log-shipper-siem-mappings.html).
+        For detailed steps refer to[Configure Log Shipper SIEM Mappings](https://docs.netskope.com/en/configure-log-shipper-siem-mappings.html).
```
Comment (efd6) [inline packages/netskope/docs/README.md:38]: I'm not sure the indentation should be changed here. Also, `s/to[Configure/to [Configure/`.

File: packages/1password/_dev/build/docs/README.md [hunk 3]
```diff
@@ -34,4 +34,4 @@ Uses the 1Password Events API to retrieve information about items in shared vaul
 
 {{fields "item_usages"}}
 
-{{event "item_usages"}}
+{{event "item_usages"}}
```
Comment (efd6) [inline packages/1password/_dev/build/docs/README.md:37]: Please revert this removal of a trailing newline.

Comment (GinnyGalicia) [reply packages/1password/_dev/build/docs/README.md]: Done

File: packages/netskope/docs/README.md (hunk 2 above)
Comment (GinnyGalicia) [reply packages/netskope/docs/README.md:38]: I've moved this to a note

File: packages/netskope/_dev/build/docs/README.md (hunk 1 above)
Comment (GinnyGalicia) [reply packages/netskope/_dev/build/docs/README.md]: Resolved. I've moved this to a note

**Comment by efd6** [comment]
This needs `elastic-package build`.

File: packages/netskope/_dev/build/docs/README.md [hunk 4]
```diff
@@ -10,34 +10,41 @@ ECS fields where applicable and the remaining fields are written under
...
     1. Navigate to Home Page > Log Shipper > SIEM Mappings
     2. Add SIEM mapping for events: 
         * Add **Rule** put rule created in step 7.
         * Add **Source Configuration** put input created for Events in step 4.
         * Add **Destination Configuration**, put output created for Events in step 6.
-    For detailed steps refer [_Configure Log Shipper SIEM Mappings_](https://docs.netskope.com/en/configure-log-shipper-siem-mappings.html).
-9. *Please make sure to use the given response formats.*
 
+<DocCallOut title="Notes">For detailed steps refer to [Configure Log Shipper SIEM Mappings](https://docs.netskope.com/en/configure-log-shipper-siem-mappings.html).
+Please make sure to use the given response formats. </DocCallOut>
```
Comment (bmorelli25) [inline packages/netskope/_dev/build/docs/README.md:47]: These docCallOuts need to be removed as they're specific to our documentation build.

**Review by bmorelli25 — CHANGES_REQUESTED** [review]

**Comment by cherryleaf** [comment]
Cannot resolve

**Pull request closed without merging**